virttest/utils_v2v.py

```diff
@@ -25,7 +25,11 @@
 
 from virttest import data_dir
 from virttest import libvirt_vm as lvirt
-from virttest import ovirt, ppm_utils
+
+try:
+    from virttest import ovirt
+except ModuleNotFoundError:
+    ovirt = None
```

Comment:
hardly see one module is set to None, so maybe just logging out something

Reply:
No, the ovirt variant is used in the code. We must define the variant first.
It not, syntax error will be reported by CI.